include/sdf/Error.hh

```diff
@@ -147,6 +147,9 @@ namespace sdf
     /// \brief Merge include is unspported for the type of entity being
     /// included, or the custom parser does not support merge includes.
     MERGE_INCLUDE_UNSUPPORTED,
+
+    /// \brief Generic error type for parameters.
```

Comment:
Since the word "parameter" is so overloaded, can you add to the comment that this pertains to the values of SDFormat elements or attributes?

Reply:
Good idea, done.